lib/index.js

```diff
@@ -216,3 +222,26 @@ function _checkAccount({req, id}) {
       });
   }
 }
+
+async function _verifyTurnstileToken({token, remoteip}) {
```

Comment:
Actually, I recommend making an "authorization.js" file and putting this in there along with the helper I suggested above -- and importing those into this file. We should get some early separation of this stuff so it's easier to maintain and refactor into middleware later if we think that's a good idea, etc.